proto/collection.proto

```diff
@@ -1,177 +1,178 @@
+  string sponsor_name = 2;
+  string logo = 3;
+  string sponsor_uri = 4;
+  string label = 5;
+  string about_uri = 6;
 }
 
 message RenderingPlatformSupport {
-    string min_bridget_version = 1;
-    string uri = 2;
+  string min_bridget_version = 1;
+  string uri = 2;
 }
 
 message Article {
-    string id = 1;
-    optional string byline = 2;
-    repeated Image images = 3;
-    Links links = 4;
-    optional string kicker = 5;
-    string title = 6;
-    optional string trail_text = 7;
-    optional int32 rating = 8;
-    optional int32 comment_count = 9;
-    optional google.protobuf.Timestamp published_date = 10;
-    optional google.protobuf.Timestamp last_updated_date = 11;
-    optional MediaType media_type = 12;
-    optional google.protobuf.Duration duration = 13;
-    optional Image profile_image = 14;
-    repeated LiveEvent events = 15;
-    optional Palette palette_light = 16;
-    optional Palette palette_dark = 17;
-    optional string apple_podcast_url = 18;
-    optional string google_podcast_url = 19;
-    optional string spotify_podcast_url = 20;
-    repeated Video videos = 21;
-    optional bool is_live = 22;
-    optional string pocket_cast_podcast_url = 23;
-    optional RenderingPlatformSupport rendered_item_prod = 24;
-    optional RenderingPlatformSupport rendered_item_beta = 25;
+  string id = 1;
+  optional string byline = 2;
+  repeated Image images = 3;
+  Links links = 4;
+  optional string kicker = 5;
+  string title = 6;
+  optional string trail_text = 7;
+  optional int32 rating = 8;
+  optional int32 comment_count = 9;
+  optional google.protobuf.Timestamp published_date = 10;
+  optional google.protobuf.Timestamp last_updated_date = 11;
+  optional MediaType media_type = 12;
+  optional google.protobuf.Duration duration = 13;
+  optional Image profile_image = 14;
+  repeated LiveEvent events = 15;
+  optional Palette palette_light = 16;
+  optional Palette palette_dark = 17;
+  optional string apple_podcast_url = 18;
+  optional string google_podcast_url = 19;
+  optional string spotify_podcast_url = 20;
+  repeated Video videos = 21;
+  optional bool is_live = 22;
+  optional string pocket_cast_podcast_url = 23;
+  optional RenderingPlatformSupport rendered_item_prod = 24;
+  optional RenderingPlatformSupport rendered_item_beta = 25;
 }
 
 message Card {
-    enum CardType {
-        CARD_TYPE_ARTICLE = 0;
-        CARD_TYPE_PODCAST = 1;
-        CARD_TYPE_VIDEO = 2;
-        CARD_TYPE_CROSSWORD = 3;
-        CARD_TYPE_DISPLAY = 4;
-        CARD_TYPE_MOSTVIEWED = 5;
-    }
-    CardType type = 1;
-    Article article = 2;
-    optional bool boosted = 3;
-    optional bool compact = 4;
-    repeated Article sublinks = 5;
-    optional string html_fallback = 6;
-    optional Branding branding = 7;
-    optional bool premium_content = 8;
-    optional Palette sublinks_palette_light = 9;
-    optional Palette sublinks_palette_dark = 10;
+  enum CardType {
+    CARD_TYPE_ARTICLE_UNSPECIFIED = 0;
```

Comment:
I think this is a valid change to make. If a zero value enum is defined (e.g. in the MAPI) response then I think it's automatically excluded from the data that's actually returned to the client. Naming zero value enums as UNSPECIFIED is also defined in the style guide: https://protobuf.dev/programming-guides/style/#enums